config/checker/org.eclipse.jgit.astub

```diff
@@ -25,7 +26,7 @@ class RevWalk {
   // This comment applies everywhere in the codebase.
   // Note that both points are kind-of enforced by Checkstyle (every occurrence of "RevCommit" must be preceded with Checker's @Unique annotation),
   // but this is not perfect - for instance, it doesn't catch RevCommits declared as `var`s.
-  @Unique RevCommit next();
+  /////// @Nullable @Unique RevCommit next();
```

Comment:
If I add `@Nullable` here, then compilation fails... but IIUC `-AuseConservativeDefaultsForUncheckedCode` should take care of that as well (?); but still, `@Nullable` next to local var seems to be ignored